pycti/entities/opencti_report.py

```diff
@@ -767,7 +767,7 @@ def add_stix_object_or_stix_relationship(self, **kwargs):
             query = """
                mutation ReportEditRelationAdd($id: ID!, $input: StixRefRelationshipAddInput!) {
                    reportEdit(id: $id) {
-                        relationAdd(input: $input) {
+                        relationAdd(input: $input, commitMessage: $commitMessage, references: $references) {
```

Comment:
`$commitMessage` and `$references` need to be added in `ReportEditRelationAdd` mutation definition just above